doc/rst/source/gmt.conf.rst

```diff
@@ -944,12 +944,12 @@ PostScript Parameters
         files and in command line parameters. This allows GMT to ensure
         that the PostScript output generates the correct characters on the
         plot. Choose from Standard, Standard+, ISOLatin1, ISOLatin1+, and
-        ISO-8859-x (where x is in the ranges [1,10] or [13,15]). See
+        ISO-8859-x (where x is in the ranges [1,11] or [13,15-16]). See
```

Comment:
```suggestion
        ISO-8859-x (where x is in the ranges 1-11 or 13-16). See
```

Reply:
But there is no 14, no?

Reply:
Is this 14?

https://github.com/GenericMappingTools/gmt/blob/master/src/PSL_ISO-8859-14.h